package.json

```diff
@@ -44,10 +44,11 @@
     "aframe": "github:mozillareality/aframe#hubs/master",
     "aframe-billboard-component": "^2.0.0",
     "aframe-inspector": "^0.8.3",
-    "aframe-physics-extras": "github:mozillareality/aframe-physics-extras#bugfix/physics-collider-world",
-    "aframe-physics-system": "github:mozillareality/aframe-physics-system#93b1dfe64c00587a9d09e1a8071750e75f340815",
+    "aframe-motion-capture-components": "github:mozillareality/aframe-motion-capture-components#aframe090",
+    "aframe-physics-system": "github:mozillareality/aframe-physics-system#de74d40f71bc07de415dc0e1fc335055019cdf6a",
```

Comment:
Prefer pointing to a named branch in our other repos (#hubs/master) when possible, then following the same policy of "whatever is the head of this branch should be the code being run in hubs". If they are ever out of sync we can see that by seeing the shah in `package.lock`

Reply:
I agree with this. The old behavior of referring to a specific commit worked better with yarn IIRC, but now referring to a branch works well. (A specific commit will wind up in the lock file, and you can do `npm install github:foo/bar#baz` again to update which commit that is to the tip of the branch.)

Reply:
For some reason I thought we preferred sha's but ok I can switch it to a branch name

Reply:
Yeah, I used to prefer SHAs also back when the tools worked less well, but now branches seem to work well.